plugins/module_utils/proxmox.py

```diff
@@ -130,7 +130,7 @@ def get_nextvmid(self):
         except Exception as e:
             self.module.fail_json(msg='Unable to retrieve next free vmid: %s' % e)
 
-    def get_vmid(self, name, ignore_missing=False, choose_first_if_multiple=False):
```

Comment:
That param is not used anywhere.

Reply:
Since `module_utils` are also a public API of the collection, it's better to only remove this in a new major release. I think a short deprecation cycle is enough, so something like changing `choose_first_if_multiple=None` and adding `if choose_first_if_multiple is not None: self.module.deprecate('The choose_first_if_multiple parameter of get_vmid() is deprecated and should no longer be used as it has no effect', version='8.0.0', collection_name='community.general')` is better.